ci/Jenkinsfile2

```diff
@@ -1,354 +1,636 @@
-#!/usr/bin/groovy
-
+#! /usr/bin/groovy
 @Library('test-shared-library@1.2') _
 
 import ai.h2o.ci.Utils
 
 def utilsLib = new Utils()
 
-properties([
-        parameters([
-                booleanParam(name: 'BUILD_PPC64LE', defaultValue: true, description: '[BUILD] Trigger build of PPC64le artifacts'),
-                booleanParam(name: 'DISABLE_TESTS', defaultValue: false, description: '[BUILD] Disable tests'),
-                booleanParam(name: 'DISABLE_COVERAGE', defaultValue: false, description: '[BUILD] Disable coverage')
-        ]),
-        buildDiscarder(logRotator(artifactDaysToKeepStr: '', artifactNumToKeepStr: '', daysToKeepStr: '180', numToKeepStr: ''))
-])
-
+///////////////
+// CONSTANTS //
+///////////////
+NODE_LABEL = 'docker && !mr-0xc8'
+OSX_NODE_LABEL = 'osx'
+PPC_NODE_LABEL = 'ibm-power'
+RELEASE_NODE_LABEL = 'master'
```

Comment:
Why we cannot keep in build matrix?

Reply:
We should implement the build matrix in next iteration, WDYT?